scipy/stats/_sensitivity_analysis.py

```diff
@@ -0,0 +1,418 @@
+       in uncertainty analysis for computer models." IEEE,
+       :doi:`10.1109/ISUMA.1990.151285`, 1990.
+
+    Examples
+    --------
+    The following is an example with the Ishigami function [6]_
+
+    .. math::
+
+        Y(\mathbf{x}) = \sin x_1 + 7 \sin^2 x_2 + 0.1 x_3^4 \sin x_1,
+
+    with :math:`\mathbf{x} \in [-\pi, \pi]^3`. This function exhibits strong
+    non-linearity and non-monotonicity.
+
+    >>> import numpy as  np
+    >>> from scipy.stats import f_ishigami, sobol_indices, uniform
+    >>> rng = np.random.default_rng(1638083107694713882823079058616272161)
+    >>> indices = sobol_indices(
+    ...     func=f_ishigami, n=1024,
+    ...     dists=[
+    ...         uniform(loc=-np.pi, scale=2*np.pi),
+    ...         uniform(loc=-np.pi, scale=2*np.pi),
+    ...         uniform(loc=-np.pi, scale=2*np.pi)
+    ...     ]
+    ... )
+    >>> indices.first_order
+    array([[ 0.29563153,  0.44014138, -0.00489234]])
+    >>> indices.total_order
+    array([[0.56138789, 0.44500977, 0.24570556]])
+
+    .. note::
+
+        In the standard form, the distribution is uniform on ``[0, 1]``.
+        Using the parameters ``loc`` and ``scale``, one obtains the uniform
+        distribution on ``[loc, loc + scale]``.
+
+    It is particularly interesting because the first order indice of
+    :math:`S_{x_3} = 0` whereas its total order is :math:`S_{T_{x_3}} = 0.244`.
+    It means that higher order interactions with :math:`x_3` are responsible
+    for the difference. Almost 25% of the observed variance
+    on the QoI is due to the correlations between :math:`x_3` and :math:`x_1`,
+    although :math:`x_3` by itself has no impact on the QoI.
+
+    The following gives a visual explanation of Sobol' indices. It shows
+    scatter plots of the output with respect to each parameter. By conditioning
+    the output value by given values of the parameter
+    (black lines), the conditional output mean is computed. It corresponds to
+    the term :math:`\mathbb{E}(Y|x_i)`. Taking the variance of this term gives
+    the numerator of the Sobol' indices. Looking at :math:`x_3`, the variance
+    of the mean is zero leading to :math:`S_{x_3} = 0`. But we can further
+    observe that the variance of the output is not constant along the parameter
+    values of :math:`x_3`. This heteroscedasticity is explained by higher order
+    interactions. Moreover, an heteroscedasticity is also noticeable on
+    :math:`x_1` leading to an interaction between :math:`x_3` and :math:`x_1`.
+    On :math:`x_2`, the variance seems to be constant and thus null interaction
+    with this parameter can be supposed. This case is fairly simple to analyse
+    visually---although it is only a qualitative analysis. Nevertheless, when
+    the number of input parameters increases such analysis becomes unrealistic
+    as it would be difficult to conclude on high-order terms.
+
+    >>> import numpy as np
```

Comment:
```suggestion
```
We discussed not importing twice. I can see making an exception here, if you prefer to keep these totally independent.